x/slashing/params.go

```diff
@@ -36,18 +39,21 @@ func ParamTypeTable() params.TypeTable {
 
 // Params - used for initializing default parameter for slashing at genesis
 type Params struct {
-	MaxEvidenceAge           time.Duration `json:"max_evidence_age"`
-	SignedBlocksWindow       int64         `json:"signed_blocks_window"`
-	MinSignedPerWindow       sdk.Dec       `json:"min_signed_per_window"`
-	DoubleSignUnbondDuration time.Duration `json:"double_sign_unbond_duration"`
-	DowntimeUnbondDuration   time.Duration `json:"downtime_unbond_duration"`
-	TooLowDelUnbondDuration  time.Duration `json:"too_low_del_unbond_duration"`
-	SlashFractionDoubleSign  sdk.Dec       `json:"slash_fraction_double_sign"`
-	SlashFractionDowntime    sdk.Dec       `json:"slash_fraction_downtime"`
-	DoubleSignSlashAmount    int64         `json:"double_sign_slash_amount"`
-	DowntimeSlashAmount      int64         `json:"downtime_slash_amount"`
-	SubmitterReward          int64         `json:"submitter_reward"`
-	DowntimeSlashFee         int64         `json:"downtime_slash_fee"`
+	MaxEvidenceAge              time.Duration `json:"max_evidence_age"`
+	SignedBlocksWindow          int64         `json:"signed_blocks_window"`
+	MinSignedPerWindow          sdk.Dec       `json:"min_signed_per_window"`
+	DoubleSignUnbondDuration    time.Duration `json:"double_sign_unbond_duration"`
+	DowntimeUnbondDuration      time.Duration `json:"downtime_unbond_duration"`
+	MaliciousVoteUnbondDuration time.Duration `json:"malicious_vote_unbond_duration"`
+	TooLowDelUnbondDuration     time.Duration `json:"too_low_del_unbond_duration"`
+	SlashFractionDoubleSign     sdk.Dec       `json:"slash_fraction_double_sign"`
```

Comment:
should we need SlashFractionMaliciousVote to cap penalty？

Reply:
The malicious vote is a kind of double sign, may be we can reuse most of the logic of double sign and also the parameters here. Just simplify the feature.